.circleci/config.yml

```diff
@@ -128,6 +128,8 @@ commands:
       - run: node .circleci/modules-to-test.js | tee packages/test.list
       - run: ./_scripts/create-version-json.sh
       - run: ./_scripts/compile-backend-ts-services.sh
+      - store_artifacts:
```

Comment:
Need to store artifacts or we won't see errors that may have happened in compile-ts-services.